pkg/scheduler/framework/runtime/framework.go

```diff
@@ -642,15 +639,14 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 		}
 		status := f.runScoreExtension(ctx, pl, state, pod, nodeScoreList)
 		if !status.IsSuccess() {
-			err := fmt.Errorf("normalize score plugin %q failed with error %v", pl.Name(), status.Message())
+			err := fmt.Errorf("normalize score plugin %q failed with error %w", pl.Name(), status.AsError())
```

Comment:
let's make this error construction consistent with the previous parallelize. I'll let you decide which pattern is best for now. But avoid saying `error` in an error message.